Source/Fuse.Reactive.JavaScript/Tests/UX/Expressions.Parameter.ux

```diff
@@ -0,0 +1,32 @@
+<Panel ux:Class="UX.ParameterFunction">
+	<Router ux:Name="router" />
+	<JavaScript>
+		router.goto("foo", { param: 1447 })
+
+		function step1() {
+			router.goto("foo", {param: 900})
+		}
+
+		function step2() {
+			router.goto("bar", {param: 300})
+		}
+
+		function step3() {
+			router.goto("foo", {param: 400, bar:"yeah"})
+		}
+
+		module.exports = { step1, step2, step3 }
```

Comment:
I believe we have a perpetual plan to run the tests on iOS and Android though. Plus, test sample inevitably get copied into production code/examples.